packages/editor/src/lib/components/default-components/DefaultCanvas.tsx

```diff
@@ -497,6 +497,25 @@ function DebugSvgCopy({ id }: { id: TLShapeId }) {
 
 	if (!isInRoot || !image) return null
 
+	if (mode === 'iframe') {
```

Comment:
when/why is this iframe used?

Reply:
this is a debug utility. `img` give a more accurate/lighter-weight view of how things will be rendered, but switching to `iframe` is useful to inspect the generated CSS in the browser dev tools